.gitignore

```diff
@@ -2,3 +2,13 @@
 .vs
 .vscode
 .idea
+.Editor
+.bin
+.download
+build.log
+upm-ci~/
+testproject/.vsconfig
+MonoBleedingEdge
+testproject/tp
```

Comment:
nit but I think testproject has its own .gitignore, should we move to there?

Reply:
Sure, I'll take care of that.

Reply:
should be cleaned up now